examples/platforms/posix/README.md

```diff
@@ -0,0 +1,57 @@
+# OpenThread on POSIX Emulation Example
+
+This directory contains example platform drivers for POSIX emulation.
+
+## Build Examples
+
+```bash
+$ cd <path-to-openthread>
+$ ./bootstrap
+$ make -f examples/Makefile-posix
+```
+
+After a successful build, the `elf` files are found in
+`<path-to-openthread>/output/x86_64-apple-darwin/bin`.
+
+## 
+
+## Interact
+
+1. Spawn the process:
+
+```bash
+$ cd openthread/examples/posix/app/cli
+$ ./ot-cli 1
+```
+
+2. Type `help` for list of commands.
+
```

Comment:
Would it make sense to keep the common cli docs (command set) with the app, and make this README focus on specifics of the platform?